include/vrpn_mocap/tracker.hpp

```diff
@@ -139,6 +141,8 @@ class Tracker : public rclcpp::Node
     return pubs->at(sensor_idx);
   }
 
+  builtin_interfaces::msg::Time get_timestamp(struct timeval vrpn_timestamp);
```

Comment:
```suggestion
  builtin_interfaces::msg::Time GetTimstamp(struct timeval vrpn_timestamp);
```
Very minor code style suggestions, to make it consistent with other class methods.

Reply:
Indeed, it makes sense. I fixed the style in the latest commit